### PR TITLE
Work around podman rm A B C flake

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM
@edsantiago PTAL

**Comment by rhatdan** [comment]
This looks like there is something more then a flake going on here.

**Comment by rhatdan** [comment]
```
[+0273s] not ok 58 podman start --all - start all containers
[+0273s] # (from function `die' in file test/system/helpers.bash, line 412,
[+0273s] #  from function `run_podman' in file test/system/helpers.bash, line 220,
[+0273s] #  in test file test/system/045-start.bats, line 33)
[+0273s] #   `run_podman rm $cid_none_explicit' failed with status 125
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman rm --all --force
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman ps --all --external --format {{.ID}} {{.Names}}
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman images --all --format {{.Repository}}:{{.Tag}} {{.ID}}
[+0273s] # quay.io/libpod/testimage:20210427 aadc32e2a626
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman run -d quay.io/libpod/testimage:20210427 /bin/true
[+0273s] # 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman run -d --restart=no quay.io/libpod/testimage:20210427 /bin/false
[+0273s] # 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman run -d --restart=on-failure quay.io/libpod/testimage:20210427 /bin/true
[+0273s] # c65e345449283fb1023bb9e497cd06fddbd1d71642d0e69ddd30e633d646f666
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman run -d --restart=always quay.io/libpod/testimage:20210427 sleep 20
[+0273s] # b652faf6ab474ff541e5d58279d1142cf2f437dffa9b522cfdae4c2279505b68
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman wait 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52 c65e345449283fb1023bb9e497cd06fddbd1d71642d0e69ddd30e633d646f666
[+0273s] # 0
[+0273s] # 1
[+0273s] # 0
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman start --all
[+0273s] # 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52
[+0273s] # 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367
[+0273s] # c65e345449283fb1023bb9e497cd06fddbd1d71642d0e69ddd30e633d646f666
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman wait 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52 c65e345449283fb1023bb9e497cd06fddbd1d71642d0e69ddd30e633d646f666
[+0273s] # 0
[+0273s] # 1
[+0273s] # 0
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman rm 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367
[+0273s] # 1fe7a0422957aa28721c4f7cd7fa8160320cb28d9f9f0fcd50d3c885d2706367
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman rm 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52
[+0273s] # Error: error removing container 0c15516948c786c7d885cb254ad9f96ab62a861bb03c526b1ebbd318c6816a52 root filesystem: 1 error occurred:
[+0273s] # 	* unlinkat /home/some16420dude/.local/share/containers/storage/overlay/839faec562a20ae2bff035b5ecd962717c2e7a6205d328f2dbf41838991f131d/merged: device or resource busy
[+0273s] # [ rc=125 (** EXPECTED 0 **) ]
[+0273s] # #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
[+0273s] # #| FAIL: exit code is 125; expected 0
[+0273s] # #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[+0273s] # # [teardown]
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman pod rm --all --force
[+0273s] # $ /var/tmp/go/src/github.com/containers/podman/bin/podman rm --all --force
[+0273s] # b652faf6ab474ff541e5d58279d1142cf2f437dffa9b522cfdae4c2279505b68
[+0273s] # c65e345449283fb1023bb9e497cd06fddbd1d71642d0e69ddd30e633d646f666
```

File: test/system/045-start.bats [hunk 1]
```diff
@@ -27,7 +27,11 @@ load helpers
 
     run_podman wait $cid_none_implicit $cid_none_explicit $cid_on_failure
 
-    run_podman rm $cid_none_implicit $cid_none_explicit $cid_on_failure
+#   FIXME:  Just removing this flaking command until we can get it fixed.
```
Comment (edsantiago) [inline test/system/045-start.bats:30]: It looks like this doesn't fix the flake, so I see no reason to merge it. But if you do, could you please change to:
```
# FIXME: #10454: ...
```
Reason: that makes it possible to remove the workaround in the future

**Pull request closed without merging**